ksql-engine/src/test/java/io/confluent/ksql/function/udtf/CubeTest.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright 2019 Confluent Inc.
+ *
+ * Licensed under the Confluent Community License (the "License"; you may not use
+ * this file except in compliance with the License. You may obtain a copy of the
+ * License at
+ *
+ * http://www.confluent.io/confluent-community-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package io.confluent.ksql.function.udtf;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.is;
+
+import com.google.common.collect.Lists;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import org.junit.Test;
+
+public class CubeTest {
+
+  private Cube cubeUdtf = new Cube();
+
+  @Test
+  public void shouldCubeSingleColumn() {
+
+    Object[] args = {1};
+    List<List<Object>> result = cubeUdtf.cube(Arrays.asList(args));
+    assertThat(result.size(), is(2));
+    assertThat(result.get(0), is(Collections.singletonList(null)));
+    assertThat(result.get(1), is(Lists.newArrayList(1)));
+
+    Object[] oneNull = {null};
```

Comment:
each test should test one thing - please split this into `shouldCubeSingleColumn` and `shouldCubeSingleNullColumn` (same with the tests below)